sdk/communication/Azure.Communication.Chat/tests/ChatClients/ChatClientsLiveTest.cs

```diff
@@ -63,20 +64,23 @@ public void ThreadCGUD_ParticipantAUR_MessageGSU_NotificationT()
                 new ChatParticipant(user3)
             };
             ChatClient chatClient = CreateInstrumentedChatClient(token1);
-            ChatClient chatClient2 = CreateInstrumentedChatClient(token2);
             ChatClient chatClient3 = CreateInstrumentedChatClient(token3);
 
             //act
             #region Snippet:Azure_Communication_Chat_Tests_E2E_InitializeChatThreadClient
```

Comment:
Just noting that snippet regions are generally placed in sample classes rather than test classes.

Reply:
Right, we did this from the beginning but will be separating them when time permits
https://skype.visualstudio.com/DefaultCollection/SPOOL/_workitems/edit/2340569